content/applications/inventory_and_mrp/manufacturing/subcontracting/dropship_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,187 @@
+
+   When purchasing a product subcontracted using the Dropship Subcontractor on Order route, it is
+   necessary to adjust this date to take into account the extra time needed for the components to be
+   delivered to the subcontractor.
+
+   Since production won't begin until they receive the components, leaving the date as is results in
+   the finished product arriving *after* the date listed on the |RfQ|.
+
+Next, click :guilabel:`Confirm Order` to turn the |RfQ| into a |PO|. Doing so creates a second |RfQ|
+to purchase the components from the dropshipper and have them sent to the subcontractor.
+
+Navigate to :menuselection:`Purchase app --> Orders --> Requests for Quotation`, and select the
+|RfQ| that lists the dropshipper in the :guilabel:`Vendor` column.
+
+On the |RfQ|, the :guilabel:`Expected Arrival` field lists the date on which the subcontractor must
+receive the components, in order to deliver the finished product by the Expected Arrival date listed
+on the subcontractor |PO|.
+
+The :guilabel:`Order Deadline` field lists the latest date on which the |RfQ| can be confirmed, in
+order for the dropshipperto deliver the components to the subcontractor by the :guilabel:`Expected
+Arrival` date.
+
+Click :guilabel:`Confirm Order` to turn the |RfQ| into a |PO|, and confirm the purchase of the
+components from the dropshipper. Doing so causes a :guilabel:`Dropship` smart button to appear at
+the top of the page.
+
+Click the :guilabel:`Dropship` smart button to open the dropship order. This order can also be
+accessed from the :guilabel:`Resupply` smart button that now appears on the subcontractor |PO|.
+
+After the dropshipper has delivered the components to the subcontractor, click the
+:guilabel:`Validate` button at the top of the dropship order to confirm that the subcontractor has
+received the components.
+
+Once the subcontractor receives the components, they begin manufacturing the component, before
+delivering it to the contracting company.
+
+.. example::
+
+   Bike retailer *Mike's Bikes* works with a subcontractor — *Bike Friends* — to produce units of
+   their *Bicycle* product.
+
+   Mike's Bikes must purchase the required components from vendor Bike World, and have them
+   dropshipped to Bike Friends.
+
+   On average, Bike Friends takes three days to manufacture each bicycle, plus an addition two days
+   to deliver it to Mike's Bikes.
+
+   As a result, Mike's Bikes sets a delivery lead time of five days for bicycle manufactured by Bike
+   Friends: three days for manufacturing, plus two days for delivery.
+
+   On the Bicycle's BoM, they enter a manufacturing lead time of five days as well, to remind
+   themselves of the date that components must be delivered to the subcontractor.
+
+   On the product pages for each of the bicycle's components, they assign Bike World a delivery lead
+   time of two days. This is the amount of time it takes Bike World to dropship each component.
```

Comment:
OH I THINK I get it now? This additional lead time is the time it takes the vendor to ship the component to the subcontractor?

In that case, to clarify, can we say something like:
```suggestion
   On the product pages for each of the bicycle's components, they assign Bike World a delivery lead
   time of two days. This is the amount of time it takes Bike World to dropship each component directly to the subcontractor.
```